File: clients/android-fireback/app/src/main/java/com/fireback/modules/workspaces/WorkspaceTypeEntity.java
package com.fireback.modules.workspaces;
import com.fireback.modules.workspaces.OkayResponseDto;
import com.google.gson.Gson;
import com.fireback.JsonSerializable;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;
public class WorkspaceTypeEntity extends JsonSerializable {
    public String title;
    public String description;
    public String slug;
    public RoleEntity role;
    public static class VM extends ViewModel {
    // upper: Title title
    private MutableLiveData< String > title = new MutableLiveData<>();
    public MutableLiveData< String > getTitle() {
        return title;
    }
    public void setTitle( String  v) {
        title.setValue(v);
    }
    // upper: Description description
    private MutableLiveData< String > description = new MutableLiveData<>();
    public MutableLiveData< String > getDescription() {
        return description;
    }
    public void setDescription( String  v) {
        description.setValue(v);
    }
    // upper: Slug slug
    private MutableLiveData< String > slug = new MutableLiveData<>();
    public MutableLiveData< String > getSlug() {
        return slug;
    }
    public void setSlug( String  v) {
        slug.setValue(v);
    }
    // upper: Role role
    private MutableLiveData< RoleEntity > role = new MutableLiveData<>();
    public MutableLiveData< RoleEntity > getRole() {
        return role;
    }
    public void setRole( RoleEntity  v) {
        role.setValue(v);
    }
    }
}

File: modules/fireback/codegen/ios/firebackios/modules/workspaces/WorkspaceTypeEntity.swift
class WorkspaceTypeEntity : Codable {
    private var _title: String?
    var `Title`: String? {
        set { _title = newValue }
        get { return _title }
    }
    private var _description: String?
    var `Description`: String? {
        set { _description = newValue }
        get { return _description }
    }
    private var _slug: String?
    var `Slug`: String? {
        set { _slug = newValue }
        get { return _slug }
    }
    private var _role: RoleEntity?
    var `Role`: RoleEntity? {
        set { _role = newValue }
        get { return _role }
    }
}